### DISCUSSION TITLE
Rotate sprite before doing push

### DISCUSSION THREAD

**Discussion by griffinmt** [Bodmer/TFT_eSPI#2428] [Q&A - Sprites]
I am trying to fill a circular sprite with a gradient background, but rotated about 30 degrees. Then I will fill it with other details before pushing it onto the screen. How do I do this.
would like a function such as:

spr.fillRectVGradient(angle,x,y,w,h,color1,color2);

or

spr.fillCircVGradient(angle,x,y,radius,color1,color2);

**Comment by Bodmer** [comment]
There is no built in function to do this. You can push rotated sprites to another one but this would consume a lot of memory.
Options:
1. Create a new function to do this (hint: alphaBlend function will mix 2 colours)
2. Create an image bitmap using a paint program
